tests/test.sh

```diff
@@ -159,6 +159,11 @@ sleep 3
 printf "${GRN}### Validating Rosetta gateway implementation...${OFF}\n"
 go run ./check-prep
 ./rosetta-cli --configuration-file rosetta-cli-config.json check:data --end 42
+{
+  # We'll cause a sigpipe on this process, so ignore the exit status.
+  # The downstream awk will exit with nonzero status if this test actually fails without confirming any transactions.
+  ./rosetta-cli --configuration-file rosetta-cli-config.json check:construction || true
+} | awk '{ print $0 }; $1 == "[STATS]" && $4 > 0 { confirmed = 1; exit }; END { exit !confirmed }'
```

Comment:
Maybe we could wait for some more transactions to be confirmed instead of just one (`$4 > 0`)?

Reply:
to answer a question of how, then yes that's how we could do that.

currently we're using this test for a few things: (1) correctness of transaction hash, (2) that a submitted transaction can go all the way and back to the gateway, and (3) that our modeling in operation sequences is correct. these are all tested in the first round trip.